includes/Utilities/Shipment.php

```diff
@@ -0,0 +1,674 @@
+			'UNITED_DELIVERY_SERVICE_LTD'                       => 'United Delivery Service, Ltd',
+			'UPS_FREIGHT'                                       => 'UPS Freight',
+			'UPS_MAIL_INNOVATIONS'                              => 'UPS Mail Innovations',
+			'VIETNAM_POST'                                      => 'Vietnam Post',
+			'VIETNAM_POST_EMS'                                  => 'Vietnam Post EMS',
+			'VIETTELPOST'                                       => 'ViettelPost',
+			'WAHANA'                                            => 'Wahana',
+			'WANBEXPRESS'                                       => 'WanbExpress',
+			'WEDO_LOGISTICS'                                    => 'WeDo Logistics',
+			'WEPOST_LOGISTICS'                                  => 'WePost Logistics',
+			'WHISTL'                                            => 'Whistl',
+			'WISELOADS'                                         => 'Wiseloads',
+			'WISE_EXPRESS'                                      => 'Wise Express',
+			'WISHPOST'                                          => 'WishPost',
+			'WNDIRECT'                                          => 'wnDirect',
+			'XDP_EXPRESS'                                       => 'XDP Express',
+			'XDP_EXPRESS_REFERENCE'                             => 'XDP Express Reference',
+			'XEND_EXPRESS'                                      => 'Xend Express',
+			'XL_EXPRESS'                                        => 'XL Express',
+			'XPOSTPH'                                           => 'Xpost.ph',
+			'XPRESSBEES'                                        => 'XpressBees',
+			'XQ_EXPRESS'                                        => 'XQ Express',
+			'YAKIT'                                             => 'Yakit',
+			'YAMATO_JAPAN'                                      => 'Yamato Japan',
+			'YANWEN'                                            => 'Yanwen',
+			'YODEL_DOMESTIC'                                    => 'Yodel Domestic',
+			'YODEL_INTERNATIONAL'                               => 'Yodel International',
+			'YRC'                                               => 'YRC',
+			'YTO_EXPRESS'                                       => 'YTO Express',
+			'YUNDA_EXPRESS'                                     => 'Yunda Express',
+			'YUN_EXPRESS'                                       => 'Yun Express',
+			'ZEPTOEXPRESS'                                      => 'ZeptoExpress',
+			'ZINC'                                              => 'Zinc',
+			'ZJS_INTERNATIONAL'                                 => 'ZJS International',
+			'ZTO_EXPRESS'                                       => 'ZTO Express',
+			'ZYLLEM'                                            => 'Zyllem',
+		];
+
+		/**
+		 * @see https://docs.woocommerce.com/document/shipment-tracking/?_ga=2.193545197.1907176699.1599852448-1652361994.1593116723&_gac=1.227990895.1596051708.Cj0KCQjwvIT5BRCqARIsAAwwD-QunfzzbbyhGEEVmH4kdtPM1uID0g72ial0LcVu0eGdkLBcv_okGwcaAjdGEALw_wcB#section-5
```

Comment:
I think we can drop the tracking parameters:

```suggestion
		 * @see https://docs.woocommerce.com/document/shipment-tracking/#section-5
```

Reply:
Yeah, that's a lot better, thanks!